### DISCUSSION TITLE
Logout headers question

### DISCUSSION THREAD

**Discussion by iRay** [go-pkgz/auth#202] [Q&A]
Hi
I'm not sure whether it's relevant here cause it's related to an old FireFox bug.
When `LogoutHandler()` is called under the hood it calls `p.JwtService.Reset(w)`, that "expires" cookies but doesn't set header's content type.

Firefox parses the response as XML and finding no root element, shows the next error message:
_"XML Parsing Error: no root element found"_

Setting content-type header in `Reset()` fixes the error:
`w.Header().Set("Content-Type", "application/json; charset=utf-8")`

But I'm not sure whether it's a good idea to set content type here.
Here is a [discussion](https://stackoverflow.com/questions/46487220/xml-parsing-error-no-root-element-found-location-in-console-ff) on stackoverflow.
What do you think?

**Comment by paskal** [comment]
I see no problems with changing the content-type there, current value seems to be incorrect.
